src/app/data-planes/views/DataPlaneListView.spec.ts

```diff
@@ -72,4 +81,23 @@ describe('DataPlaneListView.vue', () => {
       expect(tableHeadHtml).toContain(expectedColumnHeaders[i])
     }
   })
+
+  test('shows information of selected DPP', async () => {
+    const dppName = 'cluster-1.backend-02'
+    const wrapper = await renderComponent()
+
+    const secondTableRow = wrapper.findAll('[data-testid="data-overview-table"] tbody tr')[1]
+    expect(secondTableRow.html()).toContain(dppName)
+
+    expect(secondTableRow.element.classList.contains('is-selected')).toBe(false)
+    expect(wrapper.find('[data-testid="data-plane-proxy-title"]').html()).toContain('backend')
+    expect(window.location.search.includes('name=backend'))
+
+    // The click event listener for selecting a table row is unfortunately added to table cells instead of rows so we need to trigger a click on a cell instead of the row.
```

Comment:
There is no functional error here right? This is just perf / proper implem issue right? Can we create an issue for this?

Reply:
Indeed, it’s not a functional error. It’s not an error of any form really. The fact that the event is fired from table cells just makes it a bit annoying to replicate it in tests. I don’t think a change in the implementation is necessary.